figures/README.md

```diff
@@ -41,3 +42,121 @@ However, we list information about the resources, intermediate steps, and [PBTA
 | Figure | Individual script | Notes on requirements | Linked analysis modules | PBTA data files consumed |
 |--------|--------|------------------|-------------------------|-----------------------------|
 | Figure 2 | [scripts/fig2-mutational-landscape.R`](./scripts/fig2-mutational-landscape.R) | ~128MB of RAM are needed due to the run_caller_consensus_analysis-pbta.sh handling of large MAF files|[`snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers) <br> [`mutational-signatures`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/mutational-signatures) |  `pbta-snv-lancet.vep.maf.gz` <br> `pbta-snv-mutect2.vep.maf.gz` <br> `pbta-snv-strelka2.vep.maf.gz` <br> `pbta-snv-vardict.vep.maf.gz` <br> `tcga-snv-lancet.vep.maf.gz` <br> `tcga-snv-mutect2.vep.maf.gz` <br> `tcga-snv-strelka2.vep.maf.gz` |
+
+## Color Palette Usage
+
+This project has a set of unified color palettes.
+There are six sets of hex color keys to be used for all final figures.
+They are created by running `scripts/color-palettes.R` which creates 6 TSV files
+in the `figures/palettes` folder.
```

Comment:
```suggestion
There are 6 sets of hex color keys to be used for all final figures, stored as 6 TSV files
in the `figures/palettes` folder.
```
The part about the scripts is an aside that is not about _usage_ per se. I would instead put information about making alterations to the palettes in its own section that also tells you how to run the script. It can still be under `## Color Palette Usage`